modules/internal/ChapelLocale.chpl

```diff
@@ -226,6 +226,14 @@ module ChapelLocale {
   */
   inline proc locale.maxTaskPar: int { return this._value.maxTaskPar; }
 
+  /*
+    Get the number of colocales on the locale's node.
+
+    :returns: the number of colocales on the locale's node
+    :rtype: int
+  */
+  inline proc locale.numColocales: int { return this._value.numColocales; }
```

Comment:
This should be unstable pending design discussion and stabilization. Also make sure there is a test with `--warn-unstable`

Reply:
Done.